src/test/java/net/datafaker/UniqueTest.java

```diff
@@ -0,0 +1,141 @@
+package net.datafaker;
+
+import net.datafaker.service.RandomService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import static java.util.Arrays.asList;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
please use assertJ assertions instead of junit's

Reply:
This should be resolved now.